lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v2.py

```diff
@@ -19,9 +19,9 @@
 from easydict import EasyDict
 
 from executor.train import start_training
+from blueoil.data_processor import Sequence
```

Comment:
Why do you change the line?

Reply:
When I fix conflict, This line changed accidentally.

Reply:
@suttang change reverted.